src/Makefile.am

```diff
@@ -33,7 +33,7 @@ lib_LTLIBRARIES = libmeshlink.la
 
 pkginclude_HEADERS = meshlink++.h meshlink.h
 
-libmeshlink_la_LDFLAGS = -export-symbols-regex '^(meshlink_|devtool_)'
+libmeshlink_la_LDFLAGS = -export-symbols meshlink.sym
```

Comment:
My patch that fixes `meshlink_errno` on Android was not completely correct. It works if you run `./configure` in the root source directory of MeshLink, but it doesn't work if you build in a different directory. The fix is simple, just change this line to:
```
libmeshlink_la_LDFLAGS = -export-symbols $(srcdir)/meshlink.sym
```